api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpLostReport.java

```diff
@@ -145,6 +145,9 @@ public ReportDefinition constructReportDefinition() {
         addColumn(dsd, "HIV Enrolled Date", hivPatientData.getEnrollmentDate());
         addColumn(dsd, "ART Start Date", hivPatientData.getArtStartDate());
         addColumn(dsd, "Current Regimen", hivPatientData.getCurrentRegimen());
+        addColumn(dsd, "VL Quantitative",  hivPatientData.getCurrentViralLoad());
+        addColumn(dsd, "VL Date", hivPatientData.getViralLoadDate());
+        addColumn(dsd,"Model", hivPatientData.getDSDMModel());
```

Comment:
@solemabrothers Call this "DSDM Model" instead of just model

Reply:
But @ssmusoke DSDM Model is repitition

Reply:
@solemabrothers it is repetition for clarity, as a person using this report for the first time is not sure what model you are referring to